### PR TITLE
Integrate code churn metrics into tables.

### PR DESCRIPTION
**Pull request by uhafner**

### CODE REVIEW

File: plugin/pom.xml [hunk 1]
```diff
@@ -29,7 +29,7 @@
 
     <analysis-model-api.version>9.3.2</analysis-model-api.version>
     <analysis-model-tests.version>${analysis-model-api.version}</analysis-model-tests.version>
-    <forensics-api-plugin.version>0.8.0-rc710.f9a5c103b329</forensics-api-plugin.version>
+    <forensics-api-plugin.version>0.8.0-rc719.d6b007b88023</forensics-api-plugin.version>
```
Comment (jglick) [inline plugin/pom.xml:32]: @uhafner you should have a non-incremental release of all dependencies prior to merging anything to `master`.

Comment (uhafner) [reply plugin/pom.xml:32]: Ah, ok. Why is that? To always find a matching version of the dependency in the update center?

Comment (jglick) [reply plugin/pom.xml:32]: Pending a safety belt in JENKINS-50804, to be absolutely sure you do not release this as is. The intent of JEP-305 is to offer well-defined artifacts for the duration of a set of matching PRs, with the dependencies converted to official versions as the upstream PRs are merged and released.

Comment (uhafner) [reply plugin/pom.xml:32]: Ok, thanks for clarification! I'm about to create a Christmas release so it will be fixed very soon...

**Pull request merged**